Add/universal login (#9038)
fishtown-analytics/docs.getdbt.com: website/snippets/login_url_note.md

:::success Logging in

> Users can sign in at [https://login.dbt.com](https://login.dbt.com) to see accounts they have access to across instances and choose where to open <Constant name="dbt_platform" />. This is the recommended entry point for most users.

Reviewer: ```suggestion
Users can sign in at [https://login.dbt.com](https://login.dbt.com) to view accounts they have access to across instances and choose where to open <Constant name="dbt_platform" />. This is the recommended entry point for most users.
```

> For SSO with your identity provider using your account **login URL slug**, you can also use the following URL, replacing `LOGIN-SLUG` with the value used in the previous steps and `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan:

Reviewer: ```suggestion
For SSO through your identity provider, you can also use the following URL format with your account login URL slug. Replace `LOGIN_SLUG` with the value from the previous steps and `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan:
```
